src/test/java/core/basesyntax/service/CsvFileReaderTest.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service;
+
+import core.basesyntax.service.impl.CsvFileReader;
+import java.util.Collections;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CsvFileReaderTest {
+    private static final FileReader FILE_READER = new CsvFileReader();
+
+    @Test
+    public void readFromFile_Ok() {
```

Comment:
I agree with your colleague on method names
can we add the actual method name everywhere?